android/build.gradle

```diff
@@ -1,68 +1,57 @@
-buildscript {
-  ext.safeExtGet = {prop, fallback ->
-    return rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
-  }
-  
-  repositories {
-    google()
-    jcenter()
-  }
-
-  dependencies {
-    classpath rootProject.ext.has('gradleBuildTools') ? rootProject.ext.get('gradleBuildTools') : 'com.android.tools.build:gradle:3.4.2'
-    classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1'
-    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${safeExtGet('kotlinVersion', '1.3.61')}"
-  }
+plugins {
+    id "com.android.library"
+    id "org.jetbrains.kotlin.android"
```

Comment:
I can see these two plugins are "duplicated" in the `settings.gradle.kts` file. You explained the purpose of that file in the description.

My guess is that this syntax "applies" the plugins, while `settings.gradle.kts` "configures" them?

Reply:
That's correct. That separation allows us to include this module in another project and get the plugin version configuration from that project.